src/Microsoft.AspNetCore.Mvc.Localization/LocalizedHtmlString.cs

```diff
@@ -33,8 +33,20 @@ public LocalizedHtmlString(string name, string value)
         /// <param name="value">The string resource.</param>
         /// <param name="isResourceNotFound">A flag that indicates if the resource is not found.</param>
         public LocalizedHtmlString(string name, string value, bool isResourceNotFound)
-            : this(name, value, isResourceNotFound, arguments: EmptyArray<string>.Instance)
```

Comment:
Do we need to change the ctor above too? Just so that different code paths don't have different initialized values for the same thing.